index.html

```diff
@@ -101,7 +101,7 @@ <h2 class="project-title">Tonic</h2>
                         <li class="tag">css</li>
```

Comment:
Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and make the write professional commit messages.